runtime/shared_common/OSCachemmap.cpp

```diff
@@ -1663,13 +1666,19 @@ SH_OSCachemmap::syncUpdates(void* start, UDATA length, U_32 flags)
 {
 	PORT_ACCESS_FROM_PORT(_portLibrary);
 
-	IDATA rc;
+	IDATA rc = 0;
```

Comment:
This is C++ code, just move the declaration to line 1672 where `rc` gets its real value.

Reply:
Fixed.